src/sparse/vec.rs

```diff
@@ -834,6 +834,51 @@ where N: Copy + Num + Default,
     }
 }
 
+impl<N, IS1, DS1, IS2, DS2> Add<CsVecBase<IS2, DS2>>
+for CsVecBase<IS1, DS1>
+where N: Copy + Num,
+      IS1: Deref<Target=[usize]>,
```

Comment:
`IS1` could be deref on a generic Index type, as in https://github.com/vbarrielle/sprs/blob/master/src/sparse/vec.rs#L814-L823

Reply:
Will fix it with the PR, while I am at it ;)